clean up handle web API errors (#21121)
dotnet/AspNetCore.Docs: aspnetcore/web-api/handle-errors.md

    [!code-csharp[](handle-errors/samples/3.x/Filters/HttpResponseExceptionFilter.cs?name=snippet_HttpResponseExceptionFilter)]

>     In the preceding filter, 10 is subtracted from the maximum integer value. Subtracting 10 allows other filters to run at the end of the pipeline.

Reviewer: ```suggestion
    The preceding filter specifies an `Order` of the maximum integer value minus 10. This allows other filters to run at the end of the pipeline.
```
